intranet/apps/dataimport/management/commands/import_users.py

```diff
@@ -0,0 +1,124 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from django.db.models import Q
+import csv
+import json
+import sys
+
+class Command(BaseCommand):
+    help = "Import Fake People"
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('data_fname')
+       
+
+    def clean(self, name):
+        return name.replace(" ", "").replace("-","").replace("_","").replace("'", "")
+    
+    def handle(self, *args, **kwargs):
+        with open(kwargs["data_fname"]) as f:
+            data = json.load(f)
+
+        
+        counselor_data = data["counselors"]
```

Comment:
This should be moved to the below `for` loop since we aren't using `counselor_data` for anything else.